userInterface.js

```diff
@@ -1,27 +1,43 @@
 var blessed = require('blessed');
+var colorPal = require('./themes.js');
+var supportsColor = require('supports-color');
 
+// Colors
+var colorSupportLevel = supportsColor ? supportsColor.level : 0;
+var defaultThemesByColorLevel = {
+    3: colorPal.default,
+    2: colorPal.default, // Not tested yet
+    1: colorPal.tty16Default,
+    0: colorPal.tty1Default // Can be "tested" by passing --no-color argument
+};
+
+var theme = defaultThemesByColorLevel[colorSupportLevel];
+var colors = theme.palette;
```

Comment:
I think that if the logic for choosing the color level were in its own module, the module should be able to just return the theme object. That way we don't have to deal with importing/choosing themes here (i.e. the module would just export what the `colors` variable is right now).

If you prefer to use constants, I think that's okay. I also think it would work to just use the independent flags right now similar to the first usage example they have on https://www.npmjs.com/package/supports-color. If people wanted to create themes, you would have to check what color level the terminal supports anyway.